core/src/com/metalrain/stocksimulator/state/GameState.java

```diff
@@ -316,5 +324,13 @@ private SerializedEntityList toSerializedEntityArray(ImmutableArray<Entity> enti
         for (Entity e:entities) l.list.add(new SerializedEntity(e));
         return l;
     }
+    private String getRandomStockName() {
+        if (random.nextBoolean()) {
+            return(letters[random.nextInt(26)] + letters[random.nextInt(26)] + letters[random.nextInt(26)]);
```

Comment:
To generate a random letter, without a array backing

char randomLetter() { return (char)(random.nextInt(26)+'a'); }